packages/cli/src/Version.ts

```diff
@@ -74,29 +76,47 @@ export class Version implements Command {
 
     const platform = await getPlatform()
     const cliQueryEngineBinaryType = getCliQueryEngineBinaryType()
-    const introspectionEngine = await this.resolveEngine(BinaryType.introspectionEngine)
-    const migrationEngine = await this.resolveEngine(BinaryType.migrationEngine)
-    // TODO This conditional does not really belong here, CLI should be able to tell you which engine it is _actually_ using
```

Comment:
Unsure what it was about

Reply:
Found it, it was old: https://github.com/prisma/prisma/pull/7813/files#diff-10c45fc842f5a53e6a551a6103dfd55323d64393bd4fbbcc7ddcea546417d7e0